db/migrate/20240213175114_add_shared_sessions_to_client_config.rb

```diff
@@ -0,0 +1,5 @@
+class AddSharedSessionsToClientConfig < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured { add_column :client_configs, :shared_sessions, :boolean, default: false, null: false }
```

Comment:
🗣️ I think this is good to go, since we **should** be using a version of [Postgres more recent than 10](https://github.com/ankane/strong_migrations?tab=readme-ov-file#adding-a-column-with-a-default-value), but I'm not sure what version we are actually running.

Reply:
Probably a good idea to update the [client config docs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Products/Sign-In%20Service/configuration/client_config.md) once we land this feature.